.github/workflows/pack-debian.yml

```diff
@@ -37,13 +36,16 @@ jobs:
     steps:
       - name: Checkout sources
         uses: actions/checkout@v4
+      - name: Check the tag consistency with the source version
+        if: github.event_name == 'push'
```

Comment:
Why not rather `if: github.ref_type == 'tag'`? This is what I had in mind...

Reply:
Ok & Done.